src/pages/Animetoast/main.ts

```diff
@@ -0,0 +1,75 @@
+import { pageInterface } from '../pageInterface';
+import { SyncPage } from '../syncPage';
+
+export const Animetoast: pageInterface = {
+  name: 'Animetoast',
+  domain: 'https://www.animetoast.cc',
+  languages: ['German'],
+  type: 'anime',
+  isSyncPage(url: string): boolean {
+    return $('.entry-title.light-title').text() != null && url.includes('link=');
+  },
+  isOverviewPage(url: string): boolean {
+    return $('.entry-title.light-title').text() != null && !url.includes('link=');
+  },
+  sync: {
+    getTitle(url: string): string {
+      return $('.entry-title.light-title')
+        .text()
+        .replace(/(Ger Sub|Ger Dub)/gi, '')
+        .trim();
+    },
+    getIdentifier(url: string): string {
+      return url.replace(/(https|http):\/\//gi, '').split('/')[1];
+    },
+    getOverviewUrl(url: string): string {
+      return `${Animetoast.domain}/${Animetoast.sync.getIdentifier(url)}`;
+    },
+    getEpisode(url: string): number {
+      const matching = $('.current-link').text().match(/(\d+)/i);
+      return matching ? parseEpisode(matching) : NaN;
```

Comment:
please move the regex and condition into the paresEpisode. Just the $('.current-link').text() should be passed to the function.